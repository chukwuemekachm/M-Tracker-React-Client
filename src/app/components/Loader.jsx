import React from 'react';
import loadingSvg from '../assets/svg/loading.svg';
import '../assets/css/loader.css';

const Loader = () => (
  <div className="loader-container">
    <div className="loader">
      <img src={loadingSvg} alt="loader" />
    </div>
  </div>
);

export default Loader;
