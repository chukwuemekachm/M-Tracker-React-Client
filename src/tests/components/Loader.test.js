import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../app/components/Loader';

describe('Loader component', () => {
  it('should render without errors', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper.find('div')).toHaveLength(2);
    expect(wrapper.find('img')).toHaveLength(1);
    expect(wrapper).toMatchSnapshot();
  });
});
