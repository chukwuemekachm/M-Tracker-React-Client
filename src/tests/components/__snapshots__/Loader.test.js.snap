// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render without errors 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    <img
      alt="loader"
      src={null}
    />
  </div>
</div>
`;
